### ISSUE TITLE
Cannot create Entity Relationship diagrams using mermaid.js syntax.

### ISSUE THREAD

**Issue by cnk** [notable/notable#1525]
- **OS Version**: Mac OS Catalina (10.15.7)
- **Notable Version**: 1.8.4 (currently the latest version)

### Current behavior

Start a new diagram and copy one of the [examples from the Mermaid site](https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram) into a fenced block, e.g.
```
    ```mermaid
    erDiagram
        CUSTOMER ||--o{ ORDER : places
        ORDER ||--|{ LINE-ITEM : contains
        CUSTOMER }|..|{ DELIVERY-ADDRESS : uses
    ```
```

### Expected behavior

In split view mode, I expected to see a diagram on the view half. Instead I see the following error message (in red):

`[mermaid error: Parse error on line 1: erDiagram CUSTOM ^ Expecting 'NEWLINE', 'SPACE', 'GRAPH', got 'ALPHA']
`
I have tried fooling around with the spacing, tabs vs spaces, etc. But can't find any combination that works.

**Comment by leopoletti** [comment]
Hello

I have the same kind of issue:

```
```mermaid
flowchart TD
    A[Start] --> B{Is it?}
    B -->|Yes| C[OK]
    C --> D[Rethink]
    D --> B
    B ---->|No| E[End]
```
```
and get the following error:
[mermaid error: Parse error on line 1: flowchart TD A[S ^ Expecting 'NEWLINE', 'SPACE', 'GRAPH', got 'ALPHA'

Notable version: 1.8.3
OS: windows 7

**Comment by edno** [comment]
Hello

Unfortunately, this is still an issue with version 1.8.4. Any suggestion?

Notable version: 1.8.4
OS: Linux (deb)

**Comment by MMK21Hub** [comment]
It looks like this issue is no longer present in the v1.9 betas.

If you're not already using them - I'd recommend switching over because they contain the last ~2 years of development, including lots of new features and bugfixes.

The latest beta release is [v1.9.0-beta.10](https://github.com/notable/notable-insiders/releases/tag/v1.9.0-beta.10).

![Screenshot showing a working diagram](https://github.com/notable/notable/assets/50421330/237d3ab2-2023-4690-a8ac-c21876e105c0)

**Comment by cnk** [comment]
Agreed. This is fixed in [v1.9.0-beta.10](https://github.com/notable/notable-insiders/releases/tag/v1.9.0-beta.10).
